src/main/java/seedu/duke/Storage.java

```diff
@@ -0,0 +1,122 @@
+package seedu.duke;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class Storage {
```

Comment:
I like how you have many methods that abstracts the file reading and writing purposes!